rotkehlchen/db/dbhandler.py

```diff
@@ -2379,6 +2392,35 @@ def get_binance_pairs(self, name: str, location: Location) -> List[str]:
             return json.loads(data[0])
         return []
 
+    def set_ftx_subaccount(self, name: str, subaccount: str) -> None:
+        """
+        This function may rise sqlcipher.DatabaseError
+        """
+        cursor = self.conn.cursor()
+        cursor.execute(
+            'INSERT OR REPLACE INTO user_credentials_mappings '
+            '(credential_name, credential_location, setting_name, setting_value) '
+            'VALUES (?, ?, ?, ?)',
+            (
+                name,
+                Location.FTX.serialize_for_db(),  # pylint: disable=no-member
+                FTX_SUBACCOUNT_DB_SETTING,
+                subaccount,
+            ),
+        )
+
+    def get_ftx_subaccount(self, name: str) -> Optional[str]:
+        cursor = self.conn.cursor()
+        result = cursor.execute(
+            'SELECT setting_value FROM user_credentials_mappings WHERE '
+            'credential_name=? AND credential_location=? AND setting_name=?',
+            (name, Location.FTX.serialize_for_db(), FTX_SUBACCOUNT_DB_SETTING),  # noqa: E501 pylint: disable=no-member
+        )
+        data = result.fetchone()
+        if data and data[0] != '' and data[0] != ' ':
```

Comment:
heh this is not a good solution. What if it's `'  '` or `'   '`?
```suggestion
        if data and data[0].strip() != '':
```

Reply:
Thank you :pray: